hypothesis/via
# Use ToastMessages from frontend-shared

Replace local copy of `ToastMessages` with the one from frontend-shared

### Testing steps

Go to http://localhost:9083/https://www.youtube.com/watch?v=EU6TDnV5osM and check toast messages look the same.
